rustls/src/tls13/key_schedule.rs

```diff
@@ -770,6 +770,23 @@ impl KeySchedule {
     }
 }
 
+/// [HKDF-Expand-Label] where the output is an AEAD key.
+///
+/// [HKDF-Expand-Label]: <https://www.rfc-editor.org/rfc/rfc8446#section-7.1>
+pub fn derive_traffic_key(
+    expander: &dyn HkdfExpander,
+    aead_alg: &dyn Tls13AeadAlgorithm,
```

Comment:
Nit: please don't move the code and change it in the same commit, this makes it hard to review the changes.